qa-tests-backend/test-images/trigger-policy-violations/most/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos
+FROM quay.io/centos/centos:stream9
```

Comment:
registry.access.redhat.com/ubi9:latest

Reply:
I tested this when making https://github.com/stackrox/stackrox/pull/12211. `csh` isn't in the default ubi9 package registries:
```
 > [3/8] RUN yum -y install csh:
0.281 Updating Subscription Management repositories.
0.281 Unable to read consumer identity
0.283
0.283 This system is not registered with an entitlement server. You can use subscription-manager to register.
0.283
0.314 Last metadata expiration check: 0:00:01 ago on Fri Aug  9 06:17:38 2024.
0.323 No match for argument: csh
0.333 Error: Unable to find a match: csh
```

```
❯ docker run --platform linux/amd64 --rm -it registry.access.redhat.com/ubi9:latest

[root@e6b615194a58 /]# yum update -y
...
Complete!

[root@e6b615194a58 /]# yum search csh
...
No matches found.
```

The same is true for ubi8.

Reply:
I tested this when making https://github.com/stackrox/stackrox/pull/12211. `csh` isn't in the default ubi9 package registries:
```
 > [3/8] RUN yum -y install csh:
0.281 Updating Subscription Management repositories.
0.281 Unable to read consumer identity
0.283
0.283 This system is not registered with an entitlement server. You can use subscription-manager to register.
0.283
0.314 Last metadata expiration check: 0:00:01 ago on Fri Aug  9 06:17:38 2024.
0.323 No match for argument: csh
0.333 Error: Unable to find a match: csh
```

```
❯ docker run --platform linux/amd64 --rm -it registry.access.redhat.com/ubi9:latest

[root@e6b615194a58 /]# yum update -y
...
Complete!

[root@e6b615194a58 /]# yum search csh
...
No matches found.
```

The same is true for ubi8.

—

However, we might be able to remove `csh` altogether (see Ross’s comment below)

Reply:
yes removed csh